build.gradle

```diff
@@ -14,6 +14,8 @@ repositories {
 
 dependencies {
     compile group: 'org.antlr', name: 'antlr4-runtime', version: '4.6'
+    compile group: 'com.neovisionaries', name: 'nv-websocket-client', version: '2.2'
```

Comment:
Netty client added in the PR https://github.com/ballerinalang/plugin-intellij/pull/422.